.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,52 @@
+**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**
+
+• [ ] Documentation update
+• [ ] Bug fix
+• [ ] New feature
+• [ ] Enhancement to an existing feature
+• [ ] Other, please explain:
+
+<!--
+    If this pull request is addressing an issue, link to the issue: "Fixes https://github.com/MarkBind/markbind/issues/XXX" or "Resolves https://github.com/MarkBind/markbind/issues/XXX"
+-->
+
+
+<!--
+    Please ensure your pull request is ready:
+    - Bug fix PR that is non-trivial **should** add a page or unit test for regression testing.
+    - Feature PR **must** add a page to the user guide for demo.
+    - Enhancement PR **should** update the user guide.
+
+    Otherwise, prefix your PR title with "[WIP]".
+-->
+
+**What is the rationale for this request?**
+
+
+**What changes did you make? (Give an overview)**
+
+
+**Provide some example code that this change will affect:**
+
+<!-- Paste the example code below: -->
+```js
+
+```
```

Comment:
Is this section applicable to this repo?

Reply:
If there are changes to attributes etc, then this would be applicable i think

Reply:
Yeah, if anyone were to change anything to the SFCs